src/main.js

```diff
@@ -4,35 +4,32 @@
 
 var coverImg = document.querySelector('.cover-image');
 var title = document.querySelector('.cover-title');
-var descriptor1 = document.querySelector('.tagline-1');
-var descriptor2 = document.querySelector('.tagline-2');
+var tagline1 = document.querySelector('.tagline-1');
+var tagline2 = document.querySelector('.tagline-2');
```

Comment:
Good decision to refactor our refactor! These names are the ones they use in Cover class